.env_file.test

```diff
@@ -10,6 +10,8 @@ DATABASE_PASS=postgres
 PROVIDER_LANGID=google
 PROVIDER_IMAGE_CLASSIFICATION=google
 ALEGRE_HOST=http://alegre:3100
-PRESTO_HOST=http://presto:8000
+PRESTO_HOST=http://presto-server:8000
```

Comment:
is there a corresponding update to docker-compose file to rename the service?

Reply:
Yes, in the `check` repo.